sdk/tests/deqp/data/gles3/shaders/linkage.test

```diff
@@ -468,6 +468,7 @@ group varying "Varying linkage"
 		case differing_interpolation_1
 			version 300 es
 			desc "varying interpolation different (flat vs. smooth)"
+			expect link_fail
```

Comment:
Expected link_fail in C++:
https://android.googlesource.com/platform/external/deqp/+/deqp-dev/data/gles3/shaders/linkage.test#468